src/ReplayCommand.cc

```diff
@@ -146,6 +154,7 @@ static bool parse_replay_arg(vector<string>& args, ReplayFlags& flags) {
     { 'x', "gdb-x", HAS_PARAMETER },
     { 0, "share-private-mappings", NO_PARAMETER },
     { 1, "fullname", NO_PARAMETER },
+    { 2, "cpu-unbound", NO_PARAMETER },
```

Comment:
`--cpu-unbound` has a `-u` short form for `rr record`. For consistency it should here too.

Reply:
Oh, right. I just modified my earlier patch, forgetting about that. Fixed.